test/ktst/CMakeLists.txt

```diff
@@ -24,4 +24,7 @@
 
 add_compile_definitions( __mod__="test/ktst" )
 
-AddExecutableTest( Test_KTST "ktsttest" "${COMMON_LIBS_READ}" )
\ No newline at end of file
+AddExecutableTest( Test_KTST "ktsttest" "${COMMON_LIBS_READ}" )
+if( RUN_SANITIZER_TESTS )
```

Comment:
Please use a specific test, like `if (TARGET Test_KTST_tsan)`, instead of testing for the variable.

Reply:
done.